webtranslate/pages/delete.py

```diff
@@ -24,7 +25,7 @@ def delete_form(userauth, prjname, lngname):
         abort(404, "Cannot delete base language!")
         return
 
-    return template("delete_translation", pmd = pmd, lng = lng)
+    return template("delete_translation", userauth = userauth, pmd = pmd, lng = lng)
```

Comment:
This all is very anti-PEP8 (the default Python coding-style). Can I suggest you run `black -l 120` (after `pip install black`) over your code some day?

Reply:
code style is for later